apps/mobile/src/lib/workers/PlaylistSyncWorker.ts

```
@@ -0,0 +1,511 @@
+		const removeOps: QueueRow[] = []
+		const reorderOps: QueueRow[] = []
+		for (const row of rows) {
+			if (row.operation === 'add_tracks') addOps.push(row)
+			else if (row.operation === 'remove_tracks') removeOps.push(row)
+			else if (row.operation === 'reorder_track') reorderOps.push(row)
+		}
+
+		const trackIds = new Set<number>()
+		for (const row of addOps) {
+			const payload = this.parsePayload(row.payload) as { trackIds?: number[] }
+			payload.trackIds?.forEach((id) => trackIds.add(id))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)